doc/contributing/contribute_code.md

```diff
@@ -13,12 +13,16 @@ Make sure you have a Java SDK available to compile the application.
 The project has moved to the [Apache Adoptium Java SDK](https://adoptium.net) 
 (previously AdoptOpenJDK) for distribution builds.
 
-z390 requires Java 1.8 or greater.
+z390 is using java 21 to build the package but code must support Java 1.8.
+
+To ensure alignment with other developers, use of a Java SDK 21 or greater
+is recommended. At a minimum, you will require an SDK of 9 or greater 
```

Comment:
"SDK of 9"  -- > "SDK version 8"

Reply:
See comment above - the javac command now uses --release 8 rather than -source and -target which are now deprecated. The --release option is only available in Java 9 and above.